sdk/search/Azure.Search/src/Options/IndexDocumentsOptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Search
+{
+    /// <summary>
+    /// Options for <see cref="SearchIndexClient.IndexDocumentsAsync(Models.IndexDocumentsBatch{Models.SearchDocument}, IndexDocumentsOptions, System.Threading.CancellationToken)"/>.
+    /// </summary>
+    public class IndexDocumentsOptions : SearchRequestOptions
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether to throw an exception on
+        /// any individual failure in the batch of document write operations.
+        ///
+        /// Set this to true if you're not inspecting the results of the Index
+        /// Documents action.
+        /// </summary>
+        public bool ThrowOnAnyError { get; set; } = false;
```

Comment:
This is the pattern we've had for Track 2 batching so far as discussed above.  Since the exception doesn't have failure details, we don't want folks to do that by default.